targets/TARGET_RENESAS/TARGET_VK_RZ_A1H/gpio_irq_api.c

```diff
@@ -71,6 +71,7 @@ static const PinMap PinMap_IRQ[] = {
 #else
 static const PinMap PinMap_IRQ[] = {
     {P9_1,  IRQ0, 4},
+    {P7_8,  IRQ1, 8},
```

Comment:
I don't know how, if I make another pull request basically the two PR will be the same but with different headlines.